server/modules/form/router.js

```diff
@@ -5,14 +5,14 @@ import middleware from './middleware';
 
 const router = express.Router();
 
-router.use([middleware.setLocals, middleware.noCache])
+router.use([middleware.noCache]);
```

Comment:
I'm curious... why did you remove it from here given it's used in every route?

Reply:
Due to the way express runs its middleware chains, it was running before the route was matched - meaning it didn't have access to the path and hence no path parameters. It now has to check the path params for the `marketingName` too so it needs to on a "matching" route.